docs/en/pid-eaa-data-model.rst

```diff
@@ -126,13 +133,13 @@ The ``verification`` claim is a JSON structure with all the following mandatory
       - **Description**
       - **Reference**
     * - **trust_framework**
-      - [NSD]. MUST be set to eidas
+      - [NSD]. For PID credential it MUST be set to ``eidas``. For (Q)EAA it MUST be set to ``eidas2``.
```

Comment:
the trust framework is the same for both type 1 and type 2, please use `eidas` only and any other information about the credential type should be defined elsewhere

Reply:
For PID we use current eidas framework (CIE/SPID). For (Q)EAAs we use new eidas framework (PID). We refer to the latter as eidas2. This is not related to type 1 or type 2. If you want, you can make a proposal, but we shouldn't use eidas for both.

Reply:
done